docs/index.md

```diff
@@ -8,7 +8,12 @@ title: FABook
 
 ![Ui](images/Ui.png)
 
-FABook is a **desktop app for managing contacts, optimized for a financial adviser to use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, FABook can get your contact management tasks done faster than traditional GUI apps.
+**FABook** can be your dependable assistant who **reminds** you of meetings and consolidates **crucial information** like financial plans and client information right at your fingertips! <br>
+
+You can now focus on giving your full attention to your clients without having to worry about things slipping your mind.
+
```

Comment:
FABook is a dependable assistant for financial advisers by reminding them of meetings and consolidating crucial information like financial plans and client information right at their fingertips! Financial advisers now focus on giving their full attention to their clients without having to worry about things slipping their mind.